docusaurus.config.js

```diff
@@ -1,76 +1,77 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-import { themes as prismThemes } from 'prism-react-renderer';
+import { themes as prismThemes } from "prism-react-renderer";
```

Comment:
If you'd like to change coding style, let's do that in a separate commit.

Reply:
No, I did it by mistake. Let it remain the way it is.